Zend/zend_API.c

```diff
@@ -2756,6 +2756,28 @@ ZEND_API void zend_add_magic_method(zend_class_entry *ce, zend_function *fptr, z
 ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arg_info_toString, 0, 0, IS_STRING, 0)
 ZEND_END_ARG_INFO()
 
+static zend_always_inline void normalize_internal_type(zend_type *type) {
```

Comment:
```suggestion
static zend_always_inline void zend_normalize_internal_type(zend_type *type) {
```
Sorry just noticed this, could you prefix the function name?